pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -507,8 +432,240 @@ mv $(workspaces.source.path)/source/.jbs/build.sh $(workspaces.source.path)`, or
 		}
 	}
 
-	for _, i := range buildTask.Results {
-		ps.Results = append(ps.Results, tektonpipeline.PipelineResult{Name: i.Name, Description: i.Description, Value: tektonpipeline.ResultValue{Type: tektonpipeline.ParamTypeString, StringVal: "$(tasks." + BuildTaskName + ".results." + i.Name + ")"}})
+	if jbsConfig.Spec.ContainerBuilds {
+		fmt.Printf("### Running container builds \n")
+
+		// TODO: ### Note - its also possible to refer to a remote pipeline ref as well as a task.
+		resolver := tektonpipeline.ResolverRef{
+			Resolver: "git",
+			Params: []tektonpipeline.Param{
+				{
+					Name: "url",
+					Value: tektonpipeline.ParamValue{
+						Type:      tektonpipeline.ParamTypeString,
+						StringVal: v1alpha1.KonfluxBuildDefinitions,
+					},
+				},
+				{
+					// TODO: ### Currently always using 'head' of branch.
```

Comment:
I'm hard-coding `main` just below as the build-definitions repository is not tagged and everyone (I think) uses latest.